From 6065db719f4ab84c9121797bc4796d67a099c96e Mon Sep 17 00:00:00 2001
From: Oleg Rakhmanov <moonman.ca@gmail.com>
Date: Mon, 9 Mar 2015 14:39:20 -0600
Subject: [PATCH] net/aria2: update to 1.18.9; disable libuv

aria2 doesn't support libuv 1.x branch and build fails if libuv headers are present (which is the case if performing full build). The workaround as suggested by aria2 developer is to disable libuv and aria2 will fall back to (a more reliable) epoll mechanism.

Signed-off-by: Oleg Rakhmanov <orakhmanov@gmail.com>
---
 net/aria2/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/net/aria2/Makefile b/net/aria2/Makefile
index cac017a..345eb3a 100644
--- a/net/aria2/Makefile
+++ b/net/aria2/Makefile
@@ -7,12 +7,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aria2
-PKG_VERSION:=1.18.7
+PKG_VERSION:=1.18.9
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@SF/aria2
-PKG_MD5SUM:=36e92af92b4370817c577ed802546842
+PKG_MD5SUM:=aef4bfc78b612ee1374cb4abb5385b8c
 PKG_INSTALL:=1
 
 PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>
@@ -60,6 +60,7 @@ CONFIGURE_ARGS += \
 	--without-libexpat \
 	--without-libcares \
 	--without-sqlite3 \
+	--without-libuv \
 	--with-libz
 
 define Package/aria2/install
